kvm/v15.0.0/release.yaml

```diff
@@ -0,0 +1,57 @@
+# Generated with:
+# devctl release create --name v15.0.0 --provider kvm --base 14.1.2 --app cert-exporter@1.7.1 --app chart-operator@2.18.0 --app coredns@1.6.0@1.8.3 --app net-exporter@1.10.2 --component kubernetes@1.20.9 --component calico@3.19.1 --component kvm-operator@update_calico --overwrite
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v15.0.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.7.1
```

Comment:
Can we use https://github.com/giantswarm/cert-exporter/releases/tag/v1.8.0 ? Or should I go the usual requests.yaml route?